### PR TITLE
[FEAT] Hacktoberfest demo: Add 2021 theme

### PR DESCRIPTION
**Pull request by csouchet**
**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/add_2021_theme/examples/index.html

### CODE REVIEW

**Comment by csouchet** [comment]
## Dark theme
### 
image

### 
image

## Light theme
### 
image

**Comment by tbouffard** [comment]
### Final decision

**Dark**

image

**Light**

image

**Review by tbouffard — APPROVED** [review]
Nice and easy to add the new 2021 theme
⚠️ the preview displayed in the examples html home could be updated to show the new 2021 theme.

File: demo/hacktoberfest-custom-themes/index.html [hunk 1]
```diff
@@ -61,13 +61,13 @@ <h2>Hacktoberfest diagram</h2>
...
-                    <div class="column col-6">
+                    <div class="column col-4">
                         <div>
                             <h5>Project name</h5>
                             <input id="project-name-input" type="text" value="BPMN Visualization">
                         </div>
                     </div>
-                    <div id="choose-use-case-panel" class="column col-6">
+                    <div id="choose-use-case-panel" class="column col-4">
                         <h5>Render</h5>
```
Comment (tbouffard) [inline demo/hacktoberfest-custom-themes/index.html:71]:
```suggestion
                        <h5>Theme</h5>
```

**Pull request merged**